docs/configuration/integrations/integrations-next/app-o11y-receiver-config.md

```diff
@@ -0,0 +1,118 @@
++++
+title = "app_o11y_config"
++++
+
+# app_o11y_receiver_config
```

Comment:
nit: Not a lot of people know what `o11y` is, I think we should pick a different name. Maybe something like frontend_agent_receiver?

Reply:
I'll leave it up to you, I just would prefer the name to not have `o11y` in it :)